Misc/NEWS.d/next/Library/2019-04-09-22-40-52.bpo-36575.Vg_p92.rst

```diff
@@ -0,0 +1,4 @@
+The ``_lsprof`` module now uses internal timer same to ``time.perf_counter()`` by default.
+``gettimeofday(2)`` was used on Unix.  New timer has better resolution on most Unix
+platforms and timings are no more impacted by system clock updates since ``perf_counter()``
```

Comment:
Another English question: is "no more" correct here, or is it "no longer"? I am not sure because I always make the mistake.